openslides/core/static/js/core/base.js

```diff
@@ -808,6 +808,7 @@ angular.module('OpenSlidesApp.core', [
                                 id: value.id,
                             };
                         } else if (
+                            value.name != 'voting/prompt' &&
```

Comment:
I prefer a whitelist. Later we can add also a check against the plugin api (something where plugins can register their collection strings for this function here).